transport_plugins/jlink/RELEASE.md

```diff
@@ -3,6 +3,10 @@
 All major changes in each released version of the jlink transport plugin are
 listed here.
 
+## 2.0.0
+
+- JLink adapter support for streaming and tracing
```

Comment:
This should actually be `1.1.0` since from a public api point of view its 100% compatible.  The public API of a device adapter is defined as that it remains usable from `HardwareManager` and even though we rewrote the internals, from an outside pov it's just a feature update with support for streaming and tracing.

Reply:
Fixed.